pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -2165,6 +2168,140 @@ var _ = Describe("migratableDomXML", func() {
 	})
 })
 
+var _ = Describe("Manager helper functions", func() {
+
+	Context("getVMIEphemeralDisksTotalSize", func() {
+
+		var tmpDir string
+		var zeroQuantity resource.Quantity
+
+		BeforeEach(func() {
+			var err error
+			tmpDir, err = os.MkdirTemp("", "tempdir")
+			Expect(err).ToNot(HaveOccurred())
+			err = os.MkdirAll(tmpDir, 0666)
```

Comment:
I think this is unneeded, `os.MkdirTemp` already creates the directory.

Reply:
Fixed, thanks!